src/log.js

```diff
@@ -7,13 +7,54 @@
 
 import mozlog from 'mozlog';
 
-import { config } from './config';
+type LowerCasedLogLevel =
+  | 'trace'
```

Comment:
Why are there so many levels here? I feel that this is overly complicated to use, and I wouldn't know which one to use. Can we get away with just a few tiers?

Reply:
Ah, I just reused the ones from the underlying library.

I'd use:
* verbose for super verbose stuff
* debug for things we want to see during development
* info for things we want to see in production
* warn for recoverable errors (maybe ?)
* error for non-fatal errors
* critical for fatal errors

I can add comments explaining the levels.

`trace` seems to have a special handling, but I'm not sure really.

Reply:
I wrote this:
```js
type LowerCasedLogLevel =
  | 'trace' //     This will gather a stack automatically.
  | 'verbose' //   This is displayed only when run with LOG_LEVEL=verbose
  | 'debug' //     This is displayed in test and development environments.
  | 'info' //      This is displayed in production too.
  | 'warn' //      This is for messages more important than info but that aren't errors.
  | 'error' //     This is for non-fatal errors.
  | 'critical'; // This is for fatal errors.
```